Cannot install ShareS Server, fails during install, refuses to run.

**Describe the bug**
Cannot install ShareS Server, fails during install, refuses to run.

**To Reproduce**
Steps to reproduce the behavior:
Attempt to run install.sh
Watch it fail when attempting to download Image-ExifTool from Sourceforge.

**Expected behavior**
Installer expected to install ShareS Server, fails at downloading Image-ExifTool as URL 404's

**ShareS Version**
Latest Release

**Screenshots**
![image](https://user-images.githubusercontent.com/20131900/72479784-ff9ac800-37ba-11ea-8d8b-cacb5c54f18a.png)


**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04
 - Browser N/A
 - Version Release

**Additional context**
```Connecting to iweb.dl.sourceforge.net (iweb.dl.sourceforge.net)|192.175.120.182|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://downloads.sourceforge.net/project/exiftool/Image-ExifTool-11.80.tar.gz?download&failedmirror=iweb.dl.sourceforge.net [following]
--2020-01-15 17:10:41--  https://downloads.sourceforge.net/project/exiftool/Image-ExifTool-11.80.tar.gz?download&failedmirror=iweb.dl.sourceforge.net
Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 216.105.38.13
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-01-15 17:10:41 ERROR 404: Not Found.

tar: Image-ExifTool-11.80.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
./install.sh: line 24: cd: Image-ExifTool-11.80: No such file or directory
Can't open perl script "Makefile.PL": No such file or directory
make: *** No targets specified and no makefile found.  Stop.
make: *** No rule to make target 'test'.  Stop.
make: *** No rule to make target 'install'.  Stop.
Finished installing dependencies!
root@hornyfur-server:/home/phoenix/ShareX-Upload-Server#```

--- a/install.sh
+++ b/install.sh
@@ -19,7 +19,7 @@ if which node > /dev/null
 fi
 cd src && npm i -g pm2
 cd src && npm i
-wget https://iweb.dl.sourceforge.net/project/exiftool/Image-ExifTool-11.80.tar.gz
+wget https://exiftool.org/Image-ExifTool-11.80.tar.gz
 tar xvf Image-ExifTool-11.80.tar.gz
 cd Image-ExifTool-11.80
 perl Makefile.PL

